wondrous-app/components/Settings/Members/MembersTableRow/styles.tsx

```diff
@@ -0,0 +1,266 @@
+
+  & > div {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const StyledSelect = styled(Select)`
+  && {
+    background: ${palette.black101};
+    color: ${palette.white};
+    height: 42px;
+    width: 100%;
+    border-radius: 6px;
+    border: 1px solid transparent;
+    transition: border 0.2s ease-out;
+
+    :hover {
+      border: 1px solid ${palette.grey79};
+    }
+
+    svg {
+      color: ${palette.white};
+      transition: transform 0.2s ease-out;
+    }
+
+    p {
+      margin: 0;
+    }
+  }
+`;
+
+export const MemberRoleSelect = styled(({ className, ...props }) => (
+  <StyledSelect {...props} {...className} MenuProps={{ classes: { paper: className } }} />
+))`
+  &.MuiPaper-root {
+    background: ${palette.black101};
+    border: 1px solid ${palette.grey79};
+    width: 100%;
+    max-width: 259px;
+    color: ${palette.white};
+    transform: translateX(18%) !important;
+  }
+
+  &.MuiPaper-root > .MuiList-root {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+  }
+
+  &.MuiPaper-root > .MuiList-padding {
+    padding: 12px;
+  }
+`;
+
+export const MemberRoleSelectValueDisplay = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+export const MemberRoleSelectMenuItem = styled(MenuItem)`
+  && {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    background: ${palette.black98} !important;
+    color: ${palette.white};
+    border-radius: 4px;
+    padding: 8px;
+    transition: background 0.2s ease-out;
+    position: relative;
+
+    :hover {
+      background: ${palette.black92} !important;
+    }
+
+    ::before {
+      position: absolute;
+      inset: 0;
+      content: '';
+      display: block;
+      padding: 1px;
+      background: ${(props) =>
+        props.isSelected
+          ? `linear-gradient(89.84deg, ${palette.highlightOrange} 5.89%, ${palette.highlightPurple} 90.32%)`
+          : 'transparent'};
+      mask: linear-gradient(${palette.white} 0 0) content-box, linear-gradient(${palette.white} 0 0);
+      -webkit-mask-composite: xor;
+      mask-composite: exclude;
+      border-radius: 4px;
+      width: 100%;
+      height: 100%;
+    }
+  }
+`;
+
+export const MemberRoleSelectMenuIconWrapper = styled.div`
+  width: 16px;
+  height: 16px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 2px solid;
+  border-color: ${(props) => (props.isSelected ? palette.highlightPurple : palette.grey78)};
+  border-radius: 1000px;
+  padding: 1px;
+`;
+
+export const MemberRole = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 5px;
+  font-family: ${typography.fontFamily};
+  font-size: 13px;
+  font-weight: 500;
+  color: ${palette.white};
+  padding: 2px 7px;
+  border: 1px solid ${(props) => (props.borderColor ? props.borderColor : palette.white)};
+  border-radius: 1000px;
+`;
+
+export const MemberRoleEmoji = styled.span``;
+export const MemberRoleLabel = styled.span`
```

Comment:
you don't need styled-component, since you can write `<Typography textTransform="capitalize">`